templates/CRM/Admin/Form/Job.tpl

```diff
@@ -105,6 +105,14 @@ CRM.$(function($) {
       <td class="label">{$form.parameters.label}<br />{docURL page="Managing Scheduled Jobs" resource="wiki"}</td>
       <td>{$form.parameters.html}</td>
     </tr>
+    <tr class="crm-job-form-block-scheduled-run-date">
+        <td class="label">{$form.scheduled_run_date.label}</td>
+        <td>{$form.scheduled_run_date.html}<br />
+            <div dlass="description">{ts}Do not run this job before this date / time. The run frequency selected above will apply thereafter.{/ts}<br />
+              {ts}Leave blank to run{/ts} {if $action eq 1}{ts}as soon as possible{/ts}{else}{ts}at next run frequency{/ts}{/if}.
```

Comment:
Sentence fragments are hard (sometimes impossible) to translate.
I'd suggest doing the full sentence both times. Might feel "less efficient" to write it out twice, but will make translators happier.